### DISCUSSION TITLE
Deactivating created links

### DISCUSSION THREAD

**Discussion by chris2deuce** [JulianPrieber/littlelink-custom#877] [Feature requests]
I think it would be great if it were possible to temporarily deactivate links that have already been created. On the ‘My Links (studio/links)’ page, in addition to the existing buttons next to the links, there could be another button with which you can deactivate or hide a link. 

This would be very useful, as I occasionally want to create links for time-limited campaigns, but don't want to delete them again after they have expired in order to create them again in the future. 

I hope that other users will find this feature useful too!

**Comment by TylonHH** [comment]
+1
Or/and set a start/end date

**Comment by JulianPrieber** [comment]
Now technically possible with the block plugin system. Towards the end of the year I may be able to put this on the roadmap.

If someones interested I'd be able to work on this as a sponored feature.
